app/build.gradle

```diff
@@ -29,21 +30,27 @@ dependencies {
     implementation project(':domain')
     implementation project(':data')
 
-    implementation 'com.android.support:appcompat-v7:26.1.0'
-    implementation 'com.android.support.constraint:constraint-layout:1.0.2'
-    implementation 'com.android.support:recyclerview-v7:26.1.0'
-    // Picasso
-    implementation 'com.squareup.picasso:picasso:2.5.2'
-    //event Rx
-    implementation 'io.reactivex.rxjava2:rxandroid:2.0.1'
-    implementation 'io.reactivex.rxjava2:rxjava:2.1.7'
+
+    def generalDependencies = rootProject.ext.generalDependencies
+    def generalTestDependencies = rootProject.ext.generalTestDependencies
+    def appDependencies = rootProject.ext.appDependencies
+
+    implementation generalDependencies.rxAndroid,
+            generalDependencies.rxJava,
+            generalDependencies.appSupport
+
+    implementation appDependencies.picasso,
+            appDependencies.recyclerView,
+            appDependencies.constraintLayout
+
+    testImplementation generalTestDependencies.mockito,
+            generalTestDependencies.junit
+
     // Kotlin
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
this one should be moved to ext as well, just in case we want to add different kotlin

Reply:
done!